Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControlTypes.cs

```diff
@@ -35,7 +35,19 @@ public enum HandleType
     {
         None = 0,
         Rotation,
-        Scale
+        Scale,
+        Translation
```

Comment:
Nit: Update the summary doccomments to mention the translation type as well.

Reply:
Done.